{"name": "test_unsuccessful_login[chrome-ft-34#435;'/.]", "status": "passed", "attachments": [{"name": "log", "source": "30548974-5aee-4e63-9031-659818e23091-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_browser", "value": "'chrome'"}, {"name": "username", "value": "'ft'"}, {"name": "password", "value": "'34#435;'/.'"}], "start": 1622284246509, "stop": 1622284247063, "uuid": "58733558-e0ac-4cda-915d-5b21e42dcd38", "historyId": "4b7838c67916d5b4e988ff2db08a9fa0", "testCaseId": "1ce5752f9bf5ac0dd16171f6da2bde9d", "fullName": "test_cases.test_user_sign_in.Test_UserSignIn#test_unsuccessful_login", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "Cloud Login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('add_logs_on_failure', 'get_browser')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_user_sign_in"}, {"name": "subSuite", "value": "Test_UserSignIn"}, {"name": "host", "value": "DESKTOP-57RCRK3"}, {"name": "thread", "value": "10072-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_user_sign_in"}]}